scripts/ssb_alert_manage

```diff
@@ -0,0 +1,109 @@
+##H Daemon for Alerting Tool (CERN SSB Status Board)
+##H Usage: ssb_alert_manage <Actions>
+##H
+##H Actions:
+##H   help        help manual
+##H   start       start the service
+##H   stop        stop the service
+##H   status      output service's status
+##H
+
+
+### ***
+#The following block of code was taken from
+#https://github.com/dmwm/deployment/blob/master/exitcodes/manage#L27
+
+ME=$(basename $(dirname $0))
+TOP=$(cd $(dirname $0)/../../.. && pwd)
+ROOT=$(cd $(dirname $0)/../.. && pwd)
+LOGDIR=$TOP/logs/$ME
+COLOR_OK="\\033[0;32m"
+COLOR_WARN="\\033[0;31m"
+COLOR_NORMAL="\\033[0;39m"
+
+### ***
+
+if [ ! -d $LOGDIR ]; then 
+  echo "Log Directory '$LOGDIR' does not exists. Creating..."
+  mkdir -p $LOGDIR
```

Comment:
this will fail if you don't have permission to create LOGDIR, please check first that you have permission over there.

Reply:
Right, I have now implemented the check for `$TOP` directory. I have also put check for `mkdir` command if it returns success or not.